src/framework/ui/view/uitheme.cpp

```diff
@@ -22,23 +22,74 @@
 
 #include "uitheme.h"
 
+#include <QAbstractItemView>
 #include <QApplication>
+#include <QMenu>
 #include <QPalette>
+#include <QStyleFactory>
+#include <QStyleOption>
+#include <QToolBar>
 #include <QVariant>
 
+#include "log.h"
+#include "notation/view/widgets/radiobuttongroupbox.h"
```

Comment:
Please, remove the dependence of the framework on the notation

Reply:
That would indeed solve the linking problem in the UTests. But it is needed to check whether we have a normal group box or a RadioButtonGroupBox, because they require different metrics. But maybe I can find a different way to do that.
We could also move RadioButtonGroupBox from the notation module to something else, like uicomponents. In fact it has nothing to do with notation.

Reply:
>We could also move RadioButtonGroupBox from the notation module to something else, like uicomponents. In fact it has nothing to do with notation.

Yes, probably this is the best solution at the moment

Reply:
Found another solution. Do you like it? If not, I will just do the first idea (moving RadioButtonGroupBox).

Reply:
>Do you like it?

No :) Your last solution is less clear than the idea to move RadioButtonGroupBox to uicomponents. I don't like that we will add a dependency in the ui module on uicomponents, but at least this is a more clear solution

Reply:
Done! 
Oh, but now the test is failing again... 
I also thought of a third solution, namely checking the class name from the metaObject of the QGroupBox. Like this:
```cpp
groupBox->metaObject()->className();
```
That should work without any dependencies on RadioButtonGroupBox.

Reply:
I applied that third solution now, I guess it is the only way.